include/similie/exterior/coboundary.hpp

```diff
@@ -216,7 +297,12 @@ KOKKOS_FUNCTION coboundary_tensor_t<TagToAddToCochain, CochainTag, TensorType> c
                                                                      lower_chain.end(),
                                                                      (*j).discrete_vector()))));
                     }
-                    sil::exterior::Cochain cochain_boundary(simplex_boundary, values);
+                    // build the cochain of the boundary
+                    Cochain cochain_boundary(
+                            boundary_chain,
+                            boundary_values[elem].allocation_kokkos_view());
+                    // integrate over the cochain forming the boundary to compute the coboundary
+                    // (*i).value() = cochain_boundary.integrate(); // Cannot be used bevause CochainIterator::operator* does not return a reference
```

Comment:
```suggestion
                    // (*i).value() = cochain_boundary.integrate(); // Cannot be used because CochainIterator::operator* does not return a reference
```